README.md

```diff
@@ -31,14 +31,20 @@ You can see the packs available in your workspace by running `codeql pack ls --
 With both a CodeQL bundle and a CodeQL workspace you can create a bundle with the command:
 
 ```bash
-codeql-bundle --bundle <path-to-bundle> --output codeql-custom-bundle.tar.gz --workspace <path-to-workspace> --log INFO <packs>
+codeql-bundle --bundle <path-to-bundle> --output codeql-custom-bundle.tar.gz --workspace <path-to-workspace-file> --log INFO <packs>
 ```
 
 If the source bundle is the platform agnostic bundle then you can create platform specific bundles to reduce the size of the used bundle(s).
 The following example creates platform specific bundles for all the currently supported platforms.
 
 ```bash
-codeql-bundle --bundle <path-to-platform-agnostic-bundle> --output <path-to-bundles-dir> --workspace <path-to-workspace> --log INFO -p linux64 -p osx64 -p win64 <packs>
+codeql-bundle --bundle <path-to-platform-agnostic-bundle> --output <path-to-bundles-dir> --workspace <path-to-workspace-file> --log INFO -p linux64 -p osx64 -p win64 <packs>
+```
+
+An example of creation of a custom bundle for OSX containing a customization pack with the name `foo/cpp-customizations` would look as follows:
+
+```bash
+codeql-bundle --bundle codeql-bundle-osx64.tar.gz --output codeql-custom-bundle --workspace codeql-workspace.yml  --log INFO foo/cpp-customizations -p osx64
```

Comment:
```suggestion
codeql-bundle --bundle codeql-bundle-osx64.tar.gz --output codeql-custom-bundle --workspace codeql-workspace.yml  --log INFO foo/cpp-customizations
```

When using a platform specific bundle, there is no need to specify the platform.
We cannot create platform specific bundles from a platform specific bundle.